packages/desktop-client/src/components/accounts/MobileTransaction.js

```diff
@@ -0,0 +1,478 @@
+          style={{ flex: '1 auto', height: '100%', overflowY: 'auto' }}
+        >
+          {sections.map(section => {
+            return (
+              <Section
+                title={monthUtils.format(section.date, 'MMMM dd, yyyy')}
+                key={section.id}
+              >
+                {section.data.map((transaction, index, transactions) => {
+                  return (
+                    <Item
+                      key={transaction.id}
+                      style={{
+                        fontSize:
+                          index === transactions.length - 1 ? 98 : 'inherit'
+                      }}
+                      textValue={transaction.id}
+                    >
+                      <Transaction
+                        transaction={transaction}
+                        categories={this.props.categories}
+                        accounts={this.props.accounts}
+                        payees={this.props.payees}
+                        showCategory={this.props.showCategory}
+                        added={this.props.isNew(transaction.id)}
+                        onSelect={() => {}} //this.props.onSelect(transaction)}
+                      />
+                    </Item>
+                  );
+                })}
+              </Section>
+            );
+          })}
+        </ListBox>
+      </>
+    );
+  }
+}
+
+function ListBox(props) {
+  let state = useListState(props);
+  let listBoxRef = React.useRef();
+  let { listBoxProps, labelProps } = useListBox(props, state, listBoxRef);
+
+  useEffect(() => {
+    function loadMoreTransactions() {
+      if (
+        Math.abs(
+          listBoxRef.current.scrollHeight -
+            listBoxRef.current.clientHeight -
+            listBoxRef.current.scrollTop
+        ) < listBoxRef.current.clientHeight // load more when we're one screen height from the end
+      ) {
+        props.loadMore();
+      }
+    }
+
+    listBoxRef.current.addEventListener('scroll', loadMoreTransactions);
+
+    return () => {
+      listBoxRef.current &&
+        listBoxRef.current.removeEventListener('scroll', loadMoreTransactions);
+    };
+  }, [state.collection]);
+
+  return (
+    <>
+      <div {...labelProps}>{props.label}</div>
+      <ul
+        {...listBoxProps}
+        ref={listBoxRef}
+        style={{
+          padding: 0,
+          listStyle: 'none',
+          margin: 0,
+          overflowY: 'auto',
+          width: '100%'
+        }}
+      >
+        {[...state.collection].map(item => (
+          <ListBoxSection key={item.key} section={item} state={state} />
+        ))}
+      </ul>
+    </>
+  );
+}
+
+function ListBoxSection({ section, state }) {
+  let { itemProps, headingProps, groupProps } = useListBoxSection({
+    heading: section.rendered,
+    'aria-label': section['aria-label']
+  });
+
+  // The heading is rendered inside an <li> element, which contains
+  // a <ul> with the child items.
+  return (
+    <>
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏unnecessary wrapping `<>`